documentation/asciidoc/topics/proc_configuring_service_type.adoc

```diff
@@ -0,0 +1,33 @@
+[id='config_service_type-{context}']
+= Configuring the Service Type
+By default, {ispn_operator} creates {brandname} clusters that use
+{cacheservice} nodes. You do not need to configure a service type if you want
+to use {cacheservice}. If you want to create {datagridservice} nodes, define the
+`.spec.service.type` resource.
+
+.Procedure
+
+* Specify `DataGrid` as the value for `spec.service.type` in your
+**Infinispan** CR.
+
+[source,options="nowrap",subs=attributes+]
+----
+spec:
+  ...
+  service:
+    type: DataGrid
+----
+
+[NOTE]
+====
+You cannot change `.spec.service.type` after you create {brandname} clusters.
+
+For example, you create a cluster of {cacheservice} nodes but then want to
+change to {datagridservice} nodes. In this case you must create a new cluster
+with {datagridservice} nodes in a different {k8s} namespace.
```

Comment:
@oraNod we can also add 'delete and create from scratch option' not only different {k8s} namespace